scripts/check_ecosystem.py

```diff
@@ -294,9 +295,47 @@ async def main(*, ruff1: Path, ruff2: Path, projects_jsonl: Optional[Path]) -> N
                 print()
                 print("</p>")
                 print("</details>")
+
+                # Count rule changes
+                for line in diff_str.splitlines():
+                    # Find rule change for current line or construction
+                    # + <rule>/<path>:<line>:<column>: <rule_code> <message>
+                    matches = re.findall(r": [A-Z]{1,3}[0-9]{3,4}", line)
+                    if len(matches) == 0:
+                        # Handle case where there are no regex matches e.g.
+                        # +                 "?application=AIRFLOW&authenticator=TEST_AUTH&role=TEST_ROLE&warehouse=TEST_WAREHOUSE" # noqa: E501, ERA001
+                        # Which was found in local testing
+                        continue
+
+                    rule_code = matches[0][2:]  # Trim leading ": "
+
+                    # Get current additions and removals for this rule
+                    current_changes = rule_changes.get(rule_code, (0, 0))
+
+                    # Check if addition or removal depending on the first character
+                    if line[0] == "+":
+                        current_changes = (current_changes[0] + 1, current_changes[1])
+                    elif line[0] == "-":
+                        current_changes = (current_changes[0], current_changes[1] + 1)
+
+                    rule_changes[rule_code] = current_changes
+
             else:
                 continue
 
+        print(f"Rules changed: {len(rule_changes.keys())}")
```

Comment:
i'd only print this box if `rule_changes` is non-empty to shorten the comment a bit

Reply:
Addressed in 5400a18